Convert/Public/ConvertFrom-MemoryStreamToSecureString.ps1

```diff
@@ -0,0 +1,35 @@
+﻿<#
+    .SYNOPSIS
+        Converts a Memory Stream to a Secure String
+
+    .DESCRIPTION
+        This cmdlet converts a Memory Stream to a Secure String using a Stream Reader object.
+
+    .PARAMETER MemoryStream
+        The Memory Stream to be converted
+
+    .LINK
+        https://msdn.microsoft.com/en-us/library/system.io.memorystream.aspx
+        https://msdn.microsoft.com/en-us/library/system.io.streamreader%28v=vs.110%29.aspx
+        https://msdn.microsoft.com/en-us/library/system.security.securestring%28v=vs.110%29.aspx
+#>
+function ConvertFrom-MemoryStreamToSecureString
+{
+    param
+    (
+        [Parameter(Mandatory=$true)]
+        [ValidateNotNullOrEmpty()]
+        [System.IO.Stream]$MemoryStream
```

Comment:
To duplicate the changes to the other ConvertFrom-MemoryStream function, we should create two parameter sets. One for an actual System.IO.MemoryStream, and one for a System.IO.Stream.

The $MemoryStream parameter in the other function is a System.IO.MemoryStream object.